serverless/aws/examples/src/main/scalajs/sttp/tapir/serverless/aws/examples/LambdaApiJsExample.scala

```diff
@@ -0,0 +1,35 @@
+package sttp.tapir.serverless.aws.examples
+
+import cats.syntax.all._
+import sttp.monad.{FutureMonad, MonadError}
+import sttp.tapir._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.serverless.aws.lambda._
+import sttp.tapir.serverless.aws.lambda.js._
+
+import scala.scalajs.js.annotation._
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+object LambdaApiJsExample {
+
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+  implicit val monad: MonadError[Future] = new FutureMonad()
+
+  val helloEndpoint: ServerEndpoint[Unit, Unit, String, Any, Future] = endpoint.get
+    .in("api" / "hello")
+    .out(stringBody)
+    .serverLogic { _ => Future(s"Hello!".asRight[Unit]) }
+
+  val options: AwsServerOptions[Future] = AwsServerOptions.default[Future].copy(encodeResponseBody = false)
+
+  val route: JsRoute[Future] = AwsServerInterpreter(options).toRoute(helloEndpoint).toJsRoute
+
+  def main(event: AwsJsRequest): Future[AwsJsResponse] = route(event)
+
+  @JSExportTopLevel(name="handler")
+  val handler: scala.scalajs.js.Function2[AwsJsRequest, Any, scala.scalajs.js.Promise[AwsJsResponse]] = {
```

Comment:
can we move at least some of this to library code? And do we need to use `scala.scalajs.js.Function2` instead of e.g. a Scala `Function2`?

Reply:
* not really, the handler must be static, unless you have an idea how to workaround this?
* `scala.scalajs.js.Function2` can't be replaced with `Function2` for some reason, but can be redefined as a method

Reply:
maybe an object which extends a trait would work?

Reply:
Not possible: `Only static objects may export their members to the top level`